.eslintrc

```diff
@@ -4,7 +4,7 @@
 	"extends": ["eslint:recommended", "plugin:node/recommended"],
 	"env": {
 		"node": true,
-		"es6": true
+    "es6": true
```

Comment:
@pllearns you changed tabs to spaces here. Please revert the changes to this file.